src/table/TableBodyWrapper.jsx

```diff
@@ -71,7 +71,9 @@ const TableBodyWrapper = ({ rootElement, tableData, constraints, selectionsAPI,
                   selState={selState}
                   selDispatch={selDispatch}
                   tabIndex={-1}
-                  onKeyDown={(evt) => handleKeyPress(evt, rootElement, rowIndex + 1, columnIndex)}
+                  onKeyDown={(evt) =>
```

Comment:
Need to make sure this only runs when selections are enabled. I'm a bit split on what is best here. Either we add a check on this listener or a separate listener on `withSelections`. It is nice too keep everything on the same listener, but then all selection stuff are not contained in `withSelections`. I think it is fine though, as long as we sanity check for `selectionsEnabled` in a nice way